bundles/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/HomekitCommandExtension.java

```diff
@@ -134,10 +136,11 @@ private void listAccessories(Console console) {
         });
     }
 
-    private void printAccessory(Integer accessory_id, Console console) {
+    private void printAccessory(String id, Console console) {
         homekit.getAccessories().forEach(v -> {
             try {
-                if (v.getId() == accessory_id) {
+                if (("" + v.getId()).contains(id) || ((v.getName().get() != null)
+                        && (v.getName().get().toUpperCase().contains(id.toUpperCase())))) {
```

Comment:
This is no more a full matching but now a partial matching. Maybe this has to be mentioned in the usage ?